src/styles/projects.css

```diff
@@ -51,16 +52,17 @@
 }
 
 .project-nav-text:hover {
-    background-color: #003057;
+    background-color: #E0BB56;
```

Comment:
Forgive me if I'm mistaken, but isn't the background color supposed to be the same across all the pages? It looks like in the **projects** page, it's the same color as the header. This is a screenshot from the **Experience ** page and the background color is lighter, as is the background color across the rest of the site.
![image](https://github.com/user-attachments/assets/7fdc89d6-3083-49d0-9584-a3f5fac8f5c5)

Reply:
Amara is right, the projects page needs to be updated to have the same light blue background that you see on the rest of the pages.

Reply:
I just updated the background color! Is the color of the individual project cards the correct dark blue accent?